src/ca/idcert.rs

```diff
@@ -223,40 +265,30 @@ impl IdCert {
     ///
     /// This check assumes for now that we are always dealing with V3
     /// certificates and AKI and SKI have to match.
-    fn validate_aki(&self, issuer_key: &PublicKey) -> Result<(), ValidationError> {
+    fn verify_issuer_key(
```

Comment:
I'm surprised that Clippy doesn't offer some (possibly unreadable) smart way to replace this awkward looking if if return else return with some sort of map.

Reply:
Hm, actually, moving the `Ok(())` into an else for the if-if and removing the returns should make that more clear?